meerkat_abacus/data_management.py

```diff
@@ -924,9 +943,12 @@ def new_data_to_codes(engine=None, debug_enabled=True, restrict_uuids=None,
                                    engine=engine)
         # Main Query
         if restrict_uuids is not None:
+            all_uuids = []
+            for form_uuids in restrict_uuids.values():
+                all_uuids += form_uuids
```

Comment:
I wonder if such a thing would work.
`all_uuids = list(itertools.chain.from_iterable(restrict_uuids.values()))`
https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable

Reply:
Probably would work, but I am not sure it is that much clearer.

Reply:
Ok. It's not but I was sure it'd be faster.
But I've tested the performance and if we need a `list` type to pass later to `Column.in_` function then the original approach is almost twice faster.